CHANGES.md

```diff
@@ -21,6 +21,7 @@ To be released.
  -  Changed the behaviors of `Binary.Equals()` and `Binary.CompareTo()`
     to be more consistent.  [[#106], [#110]]
  -  Removed `ByteArrayComparer` and `FingerprintComparer`.  [[#111]]
+ -  Removed `IEquatable<IImmutableList<IValue>>` from `List`.  [[#104], [#108]]
```

Comment:
Did you intend to mention this pull request?

```suggestion
 -  Removed `IEquatable<IImmutableList<IValue>>` from `List`.  [[#104], [#112]]
```

Reply:
A mistake from copy-pasting. 😣

Reply:
Fixed.